practice-one/src/components/Button/Button.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import "./Button.css";
+import PropTypes from "prop-types";
+
+const Button = ({
+  children,
+  width,
```

Comment:
why need to pass width, height, color, background, fontSize, fontWeight, border, align?

Reply:
Because there are many different buttons in the design, splitting it up like this makes it easier for me to control its properties.